src/lib/holocene/split-button.svelte

```diff
@@ -8,26 +8,18 @@
   export let menuLabel: string;
   export let icon: IconName | undefined = undefined;
   export let id: string;
-  export let variant:
-    | 'primary'
-    | 'secondary'
-    | 'destructive'
-    | 'login'
-    | 'link' = 'primary';
-  export let thin = false;
+  export let variant: 'primary' | 'secondary' | 'destructive' | 'ghost' =
+    'primary';
   export let disabled = false;
   export let position: 'left' | 'right' = 'left';
-  export let href = '';
   export let primaryActionDisabled = false;
 </script>
 
 <MenuContainer class={$$props.class}>
```

Comment:
Wondering if we want to do something with `z-index` for the border on hover for these buttons

![Screenshot 2023-08-29 at 3 50 14 PM](https://github.com/temporalio/ui/assets/15069288/8bd5c50d-c1dd-42e0-b7d3-de47e1699de7)
![Screenshot 2023-08-29 at 3 50 34 PM](https://github.com/temporalio/ui/assets/15069288/3021b2e4-9d8d-4de1-8d15-ce7e7d6cc847)

Reply:
I'll ask Ash about this, I'm not sure what the expected behavior here is.